style.css

```diff
@@ -3,385 +3,1016 @@
-  width: 20rem;
-}
-
-.formy-btn {
-  margin-top: 1rem;
-  margin-left: -9rem;
-  padding: 1rem 2rem 1rem;
-}
-
-form {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-
-.formy-btn:hover {
-  background: skyblue;
-  color: white;
+@media only screen and (max-width: 768px) {
+  body {
+    background-image: url("./image/body.png");
+    background-size: 80%;
+    background-position: center;
+    background-position-y: 4%;
+    background-repeat: no-repeat;
+    margin-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .swapcard3 {
+    display: none;
+  }
+
+  .main {
+    display: grid;
+    grid-template-columns: auto;
+    padding-left: 2rem;
+  }
+
+  header {
+    width: 70%;
+    padding: 12px;
+    padding-top: 3rem;
+    margin-bottom: 5rem;
+    grid-template-columns: auto auto;
+    gap: 100%;
+    display: grid;
+  }
+
+  a > .sp1 {
+    flex-grow: 70%;
+    padding-left: 1%;
+    width: 75px;
+    height: 20px;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 20px;
+    color: #6070ff;
+  }
+
+  .sp2 {
+    padding-right: 1%;
+  }
+
+  .paragraph {
+    widows: 70%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .text {
+    padding-left: 15px;
+    width: 80%;
+    height: 150px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .text2 {
+    margin-top: 1%;
+    margin-bottom: 5%;
+    padding-left: 15px;
+    width: 80%;
+    height: 146px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .card001 {
+    display: grid;
+    grid-template-columns: auto;
+  }
+
+  .a {
+    display: flex;
+    padding-left: 15px;
+    align-self: initial;
+    font-weight: 500;
+    font-size: 16px;
+    text-decoration: none;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .social-media {
+    display: flex;
+    widows: 300px;
+    justify-content: space-between;
+    max-width: 150px;
+    margin-top: 3rem;
+    padding-inline-start: 15px;
+  }
+
+  li {
+    list-style-type: none;
+  }
+
+  .social-media > li > a > img {
+    height: 20px;
+    font-size: 2px;
+    margin-right: 20px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .card1 {
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    padding: 16px 16px 16px 16px;
+  }
+
+  .card2 {
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    padding: 16px 16px 16px 16px;
+  }
+
+  .card3 {
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    padding: 16px 16px 16px 16px;
+  }
+
+  .card4 {
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    padding: 16px 16px 16px 16px;
+  }
```

Comment:
- [ ] I would suggest you kindly follow the DRY (Don't Repeat Yourself) method in your project as this follows the best coding practices. Rather than having to repeat the `properties` of the cards over and over again, you can simply give the work card the same class name and use the class name to style them all.

- [ ] Currently, the design of your work cards does not look like one of the Figma templates. I would suggest you check the `alignment`, `font size`, `font-weight`, and the `font family` used.
- [ ] Kindly check the button, as well as its `text`, is not aligned perfectly

Your design 👇 

![isma err](https://user-images.githubusercontent.com/78634841/206789959-cfa37703-2ca5-4477-b4d4-48183acc73e2.png)

Figma design 👇 

![hej card](https://user-images.githubusercontent.com/78634841/206789755-b0c5f505-ac3d-474d-bbd5-1cfeac07c906.png)